Fix GKE installation instruction (#3938)
kyma-project/kyma: docs/kyma/docs/04-04-use-your-own-domain.md

### Delegate the management of your domain to Google Cloud DNS

> > **NOTE**: Google Cloud DNS setup has to be done only once per DNS zone

Reviewer: ```suggestion
> **NOTE**: Perform Google Cloud DNS setup once per DNS zone.
```

### Get the TLS certificate

> 1. Export issuer email and domain for the cluster:

Reviewer: ```suggestion
1. Export the certificate issuer email and the cluster domain as environment variables:
```

    ```
    export CERT_ISSUER_EMAIL={YOUR_EMAIL}
    export DOMAIN="$CLUSTER_NAME.$(echo $DNS_NAME | sed `s/\.$//`)"
    ```

2. Create a folder for certificates. Run:
    ```
    mkdir letsencrypt
    ```
3. Create a new service account and assign it to the **dns.admin** role. Run these commands:
    ```
    gcloud iam service-accounts create dnsmanager --display-name "dnsmanager" --project "$GCP_PROJECT"
    ```
    ```
    gcloud projects add-iam-policy-binding $GCP_PROJECT \
        --member serviceAccount:dnsmanager@$GCP_PROJECT.iam.gserviceaccount.com --role roles/dns.admin
    ```

    > **NOTE**: You don't have to create a new DNS manager service account (SA) every time you deploy a cluster. Instead, you can use an existing SA that has the **dns.admin** assigned.

4. Generate an access key for this account in the `letsencrypt` folder. Run:
    ```
    gcloud iam service-accounts keys create ./letsencrypt/key.json --iam-account dnsmanager@$GCP_PROJECT.iam.gserviceaccount.com
    ```
    

>     > **NOTE**: There is a fixed number of keys that may be generated for single Service Account. We suggest reusing keys instead of generating new one for every cluster.

Reviewer: ```suggestion
    > **NOTE**: The number of keys you can generate for a single service account is limited. Reuse the existing keys instead of generating a new key for every cluster.
```
